buildpack.toml

```diff
@@ -278,6 +278,20 @@ api = "0.7"
       type = "GPL-2.0 WITH Classpath-exception-2.0"
       uri = "https://openjdk.java.net/legal/gplv2+ce.html"
 
+  [[metadata.dependencies]]
+    cpes = ["cpe:2.3:a:oracle:graalvm:22.3.0:*:*:*:community:*:*:*", "cpe:2.3:a:oracle:jdk:21.0.0:*:*:*:*:*:*:*"]
+    id = "native-image-svm"
```

Comment:
what does svm stand for?

Reply:
Oh, lol. That was an old name for things. I think it was "substrate VM". 

https://docs.oracle.com/en/graalvm/enterprise/20/docs/reference-manual/native-image/SubstrateVM/#the-substrate-vm-project

Reply:
It might be worth revisiting this and some of the other decisions in the buildpack. Like we install a JDK, but the JDK and NIK downloads are effectively the same thing, just NIK has the additional native-image tools. Do we really need both? or could we just remove NIK and install JDK+NIK when someone wants a JDK?

That's a different discussion though.

Reply:
yeah, we need to revisit NIK / GraalVM / gu install. etc...